src/components/dialogs/SaveErrorDialog.vue

```diff
@@ -0,0 +1,98 @@
+<template>
```

Comment:
Very similar to dialog in Filings UI except I changed "filing" to "application".